build.properties

```diff
@@ -83,3 +84,13 @@ jfx.build.jdk.buildnum.min=28
 # that is supported. It must be <= jfx.gradle.version.
 jfx.gradle.version=5.3
 jfx.gradle.version.min=4.8
+
+# Toolchains
+jfx.build.linux.gcc.version.min=gcc7.3.0-OEL6.4+1.1
+jfx.build.windows.msvc.version.min=VS2017-15.5.5+1.1
+jfx.build.macosx.xcode.version.min=Xcode9.4-MacOSX10.13+1.0
+
+# Build tools
+jfx.build.cmake.version.min=3.13.3
+jfx.build.ninja.version.min=1.8.2
+jfx.build.ant.version.min=1.10.5
```

Comment:
Similar to my earlier comment, you can probably just drop the `.min` from the variable name. If you do add variables for the minimum, they might be lower than the ones we use in production (e.g., the minimum `ant` version is `1.8.2`).

Reply:
Thanks Kevin. I will drop the min suffix.